framework/packages/abstract-std/src/objects/ownership/gov_ownable.rs

```diff
@@ -39,7 +39,7 @@ pub enum GovOwnershipError {
     #[error("Cannot transfer ownership to renounced structure. use action::renounce")]
     TransferToRenounced,
 
-    #[error("Cannot change NFT ownership")]
+    #[error("Cannot change NFT ownership. Transfer the NFT to change account ownership.")]
```

Comment:
We use this error for Renounce as well! Would you prefer to have separate error for renounce then?

Reply:
I checked and it doesn't look like we do.